### PR TITLE
update _index.md for docs and fix translation

### PR DESCRIPTION
**Pull request by ethan-daocloud**
This md is index page for Documentation, not for home page

### CODE REVIEW

**Comment by tengqm** [comment]
/lgtm
Thanks.

File: content/zh/docs/_index.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
 ---
-title: Home
-weight: 5
```
Comment (idealhack) [inline content/zh/docs/_index.md]: why removing this property?

Comment (idealhack) [reply content/zh/docs/_index.md]: this is not needed. thanks.

/lgtm

Comment (ethan-daocloud) [reply content/zh/docs/_index.md]: Yes, the weight property does not exist in original en docs. thanks.

**Pull request merged**